frontend/packages/ux-editor/src/components/config/editModal/EditAutoComplete.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import type { FormComponentType } from '../../../types/global';
+import { TextField, Popover, Button } from '@digdir/design-system-react';
+
+const getLastWord = (value: string) => value.split(' ').pop();
+const stdAutocompleteOpts = [
+  'on',
+  'off',
+  'name',
+  'honorific-prefix',
+  'given-name',
+  'additional-name',
+  'family-name',
+  'honorific-suffix',
+  'nickname',
+  'email',
+  'username',
+  'new-password',
+  'current-password',
+  'one-time-code',
+  'organization-title',
+  'organization',
+  'street-address',
+  'address-line1',
+  'address-line2',
+  'address-line3',
+  'address-level4',
+  'address-level3',
+  'address-level2',
+  'address-level1',
+  'country',
+  'country-name',
+  'postal-code',
+  'cc-name',
+  'cc-given-name',
+  'cc-additional-name',
+  'cc-family-name',
+  'cc-number',
+  'cc-exp',
+  'cc-exp-month',
+  'cc-exp-year',
+  'cc-csc',
+  'cc-type',
+  'transaction-currency',
+  'transaction-amount',
+  'language',
+  'bday',
+  'bday-day',
+  'bday-month',
+  'bday-year',
+  'sex',
+  'tel',
+  'tel-country-code',
+  'tel-national',
+  'tel-area-code',
+  'tel-local',
+  'tel-extension',
+  'url',
+  'photo',
+];
+
+export interface Props {
```

Comment:
You could just use the `IGenericEditComponent` interface here (from `componentConfig.ts`), which is used in the other Edit-components.

Reply:
Thanx. Have merge conflicts... will make that change when code is "open".